dataland-frontend/src/components/resources/dataTable/cells/DocumentLinkDisplayComponent.vue

```diff
@@ -10,17 +10,17 @@
 <script lang="ts">
 import { defineComponent } from "vue";
 import {
-  type MLDTDisplayComponents,
-  type MLDTDisplayValue,
-} from "@/components/resources/dataTable/MultiLayerDataTableCells";
+  type MLDTDisplayComponentName,
+  type MLDTDisplayObject,
```

Comment:
Thats also just renaming, correct?

Reply:
Yes.
I renamed `MLDTDisplayValue` to `MLDTDisplayObject` because it does not only contain the value to display, but also the info what name the component has that shall be used to display the value.
So I thought "MLDTDisplayObject" expresses this better.

Reply:
ok, i ilke the changes. only wanted to make sure that nothing besides the naming was changed. i will resolve here